source/scss/_pf-component-card.scss

```diff
@@ -2,8 +2,15 @@
 // Base styles
 //
 
-.card {
+.pf-card::after {
```

Comment:
This is the first time I see this solution, have I mention that you should blog about these things?